tsconfig-strict.json

```diff
@@ -171,6 +187,8 @@
     "core/templates/services/schema-form-submitted.service.spec.ts",
     "core/templates/services/stateful/background-mask.service.ts",
     "core/templates/services/stateful/background-mask.service.spec.ts",
+    "core/templates/services/translations-backend-api.service.ts",
+    "core/templates/services/translations-backend-api.service.spec.ts",
```

Comment:
These files were removed with merge of https://github.com/oppia/oppia/pull/12790/files ,we can remove them from list(this is also causing type checks to fail)

Reply:
@EeshaArif This is not addressed.

Reply:
Done